public/home.js

```diff
@@ -178,13 +180,30 @@ function init_chart(activities_per_user, user_by_id) {
             amounts[activity_weekday] += activities[i].amount;
         }
 
+        function getCookie(name) {
+            var nameEQ = name + "=";
+            var ca = document.cookie.split(';');
+            for(var i=0;i < ca.length;i++) {
+                var c = ca[i];
+                while (c.charAt(0)==' ') c = c.substring(1,c.length);
+                if (c.indexOf(nameEQ) == 0) return c.substring(nameEQ.length,c.length);
+            }
+            return null;
+        }
+
+        if(getCookie('uid') == author_id) {
```

Comment:
```suggestion
        if (getCookie('uid') === author_id) {
```
I would prefer a space after if and loop statements before the brace and also a comparison with 3 `=` (as the comparison is faster and generally more accurate)